Logger/LogFactory.cs

```diff
@@ -2,10 +2,23 @@
 {
     public class LogFactory
     {
+        private string filePath { get; set; }
```

Comment:
Properties should be PascalCase 
```suggestion
        private string FilePath { get; set; }
```